src/addon/client.cpp

```diff
@@ -158,8 +158,8 @@ bool addons_client::upload_addon(const std::string& id, std::string& response_me
 			vgettext("The add-on <i>$addon_title</i> has an invalid file or directory "
 				"name and cannot be published. "
 
-				"File or directory names may not contain whitespace, control characters or any of the following characters: '&quot; * / : &lt; &gt; ? \\ | ~'. "
-				"It also may not contain '..' end with '.' or be longer than 255 characters."
+				"File or directory names may not contain '..' or end with '.' or be longer than 255 characters. "
+				"It also may not contain whitespace, control characters, or any of the following characters:\n\n' * / : &lt; &gt; ? \\ | ~"
```

Comment:
I thought the apostrophe _was_ allowed? In fact, it almost looks like you accidentally replaced `"` with `'` in the list of disallowed characters.

Reply:
Actually, it looks exactly like that. I'd have expected both to be disallowed. But really, this is all proving why the original formatting was a problem.

@Pentarctagon if you could, verify the list against the code.

Reply:
Yep, you're right, it's `"` that's not allowed rather than `'`.

Reply:
Since it's literally a single-character fix, I'll just do it myself. Thanks for confirming though.